lib/cocoapods-core/specification/root_attribute_accessors.rb

```diff
@@ -25,6 +25,12 @@ def requires_arc
           attributes_hash['requires_arc']
         end
 
+        # @return [Bool, String] The prefix_header_file value.
+        #
+        def prefix_header_file
```

Comment:
this method is specified within `root_attribute_accessor` but within `dsl.rb` the declaration of of the attribute itself does not use `root_attribute` but rather `attribute`.

Is this the right place to include this method?

Reply:
I'm not sure, but I modeled it after the `requires_arc` implementation which does the same thing.